settings.gradle.kts

```diff
@@ -8,13 +8,15 @@ pluginManagement {
 }
 
 dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
```

Comment:
This is for extra safety this `dependencyResolutionManagement` is the single source of truth for all repositories used in the project